### PR TITLE
Display branch commit status

### LINKED ISSUE #10388: Display commit status on branches list page.
As title.

### PR DESCRIPTION
**Pull request by lunny**
Fix #10388

This PR adds a status icon for every branch which has a status check for the latest commit on branch list page.

图片

### CODE REVIEW

**Comment by silverwind** [comment]
Screenshot?

**Comment by silverwind** [comment]
This table needs to be rewritten later, but I guess this position is ok until than.

**Comment by denyskon** [comment]
> Separare column in the UI may be nice, but this table needs to be rewritten later, so I guess this position is ok until then.

That's the way we do it on commits list, so I also think we can stay with it for now.

**Review by denyskon — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by yardenshoham** [comment]
I'd call this a feature

File: models/git/commit_status.go [hunk 1]
```diff
@@ -346,6 +346,53 @@ func GetLatestCommitStatusForPairs(ctx context.Context, repoIDsToLatestCommitSHA
...
+	}
+
+	results := make([]result, 0, len(commitIDs))
+
+	sess := db.GetEngine(ctx).Table(&CommitStatus{})
+
+	// Create a disjunction of conditions for each repoID and SHA pair
+	conds := make([]builder.Cond, 0, len(commitIDs))
+	for _, sha := range commitIDs {
+		conds = append(conds, builder.Eq{"sha": sha})
```
Comment (delvh) [inline models/git/commit_status.go:363]: Isn't that a `builder.In`?

**Pull request merged**